packages/wonder-blocks-accordion/src/components/accordion-section-header.tsx

```diff
@@ -0,0 +1,197 @@
+    // For internal use only.
+    isFirstSection: boolean;
+    // Whether this section is the last section in the accordion.
+    // For internal use only.
+    isLastSection: boolean;
+};
+
+const AccordionSectionHeader = (props: Props) => {
+    const {
+        header,
+        caretPosition,
+        cornerKind,
+        expanded,
+        onClick,
+        sectionContentUniqueId,
+        headerStyle,
+        tag,
+        testId,
+        isFirstSection,
+        isLastSection,
+    } = props;
+
+    const headerIsString = typeof header === "string";
+
+    const {roundedTop, roundedBottom} = getRoundedValuesForHeader(
+        cornerKind,
+        isFirstSection,
+        isLastSection,
+        expanded,
+    );
+
+    return (
+        <HeadingSmall tag={tag} style={styles.heading}>
+            <Clickable
+                aria-expanded={expanded}
+                aria-controls={sectionContentUniqueId}
+                onClick={onClick}
+                testId={testId}
+                style={[
+                    styles.headerWrapper,
+                    caretPosition === "start" && styles.headerWrapperCaretStart,
+                    roundedTop && styles.roundedTop,
+                    roundedBottom && styles.roundedBottom,
+                    headerStyle,
+                ]}
+            >
+                {() => (
+                    <>
+                        <View
+                            style={[
+                                styles.headerContent,
+                                headerIsString && styles.headerString,
+                            ]}
+                        >
+                            {headerIsString ? (
+                                <View
+                                    style={[
+                                        caretPosition === "end"
+                                            ? styles.headerStringCaretEnd
+                                            : styles.headerStringCaretStart,
+                                    ]}
+                                >
+                                    {header}
+                                </View>
+                            ) : (
+                                header
+                            )}
+                        </View>
+                        <PhosphorIcon
+                            icon={caretDown}
+                            color={Color.offBlack64}
+                            size="small"
+                            style={[
+                                caretPosition === "start"
+                                    ? styles.iconStart
+                                    : styles.iconEnd,
+                                expanded && styles.iconExpanded,
+                            ]}
+                        />
+                    </>
+                )}
+            </Clickable>
+        </HeadingSmall>
+    );
+};
+
+const activeBlue = mix(Color.offBlack32, Color.blue);
```

Comment:
**suggestion:** You could also use `tokens.color.activeBlue` directly from `wb-theming` here.

https://khan.github.io/wonder-blocks/?path=/docs/theming-tokens-color--docs

**NOTE:** The mid-term goal is to deprecate the `Color` and `Spacing` packages and instead rely on a centralized tokens object (that we are already defining as part of the theming package).